data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/RecordDispatcherImpl.java

```diff
@@ -77,6 +78,7 @@ public class RecordDispatcherImpl implements RecordDispatcher {
   private static final String KN_ERROR_DEST_EXT_NAME = "knativeerrordest";
   private static final String KN_ERROR_CODE_EXT_NAME = "knativeerrorcode";
   private static final String KN_ERROR_DATA_EXT_NAME = "knativeerrordata";
+  private static final String EKB_ERROR_PREFIX       = "kne-";
```

Comment:
@pierDipi I have no idea what is better `kne` as in Knative Error?

Or `ekb`, as in "eventing kafka broker", as that is more generic

Reply:
I like `kne-`